src/KpisServiceProvider.php

```diff
@@ -42,6 +53,30 @@ public function boot(Modules $modules, Router $router)
         $this->publishes([
             __DIR__.'/public/assets' => public_path('vendor/kpis'),
         ], 'public');
+
+        if ($this->app->runningInConsole()) {
+            $this->commands([
+                CountUniqueUser::class,
+                CountUser::class,
+                DetermineStorageUsage::class,
+            ]);
+
+            $this->app->booted(function () {
+                $schedule = app(Schedule::class);
+
+                $schedule->command(CountUniqueUser::class)
+                    ->monthlyOn(1, '0:0')
```

Comment:
This is equivalent to:
```suggestion
                    ->monthly()
```